docs/framework-dyna_trace_agent.md

```diff
@@ -0,0 +1,97 @@
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Tags</strong></td>
+    <td><tt>dyna-trace-agent=&lt;version&gt;</tt></td>
+  </tr>
+</table>
+Tags are printed to standard output by the buildpack detect script
+
+## User-Provided Service
+Users must provide their own DynaTrace service. A user-provided DynaTrace service must have a name or tag with `dynatrace` in it so that the DynaTrace Agent Framework will automatically configure the application to work with the service.
+
+The credential payload of the service may contain the following entries:
+
+| Name | Description
+| ---- | -----------
+| `server` | The DynaTrace collector hostname to connect to.   Use `host:port` format for a specific port number.
+| `profile` | (optional) The DynaTrace server profile this is associated with.   Uses `Monitoring` by default.
+
+**NOTE** Be sure to open an Application Security Group to your DynaTrace collector prior to starting your application:
+```
+$ cat security.json
+   [
+     {
+       "protocol": "tcp",
+       "destination": "dynatrace_host",
+       "ports": "9998"
+     }
+   ]
+
+$ cf create-security-group dynatrace_group ./security.json
+Creating security group dynatrace_group as admin
+OK
+
+$ cf bind-running-security-group dynatrace_group
+Binding security group dynatrace_group to defaults for running as admin
+OK
+
+TIP: Changes will not apply to existing running applications until they are restarted.
+```
+
+## Configuration
+For general information on configuring the buildpack, refer to [Configuration and Extension][].
+
+The framework can be configured by modifying the [`config/dyna_trace_agent.yml`][] file in the buildpack fork.  The framework uses the [`Repository` utility support][repositories] and so it supports the [version syntax][] defined there.
+
+| Name | Description
+| ---- | -----------
+| `repository_root` | The URL of the DynaTrace repository index ([details][repositories]).
+| `version` | The version of DynaTrace to use. This buildpack framework has been tested on 6.1.0.
+
+
+**NOTE:**  This framework does not connect to a pre-populated repository.  Instead you will need to create your own repository by:
+
+1.  Downloading the DynaTrace agent unix binary (in JAR format) to an HTTP-accesible location
+1.  Uploading an `index.yml` file with a mapping from the version of the agent to its location to the same HTTP-accessible location
+1.  Configuring the [`config/dyna_trace_agent.yml`][] file to point to the root of the repository holding both the index and agent binary
+
+Sample **`repository_root`** for dyna_trace_agent.yml (under java-buildpack/config) assuming a bosh-lite setup and a local webserver (e.g. `brew install tomcat7`) on port 8080
```

Comment:
This could just be a link to the `dyna_trace_agent.yml` config file instead of referencing it's location textually.